Dockerfile

```diff
@@ -16,6 +16,7 @@ COPY --from=build-env /src/controller ${OPERATOR}
 
 COPY --from=build-env /src/build/bin /usr/local/bin
 RUN  /usr/local/bin/user_setup
+RUN microdnf update && microdnf install curl
```

Comment:
good catch but the image still big even with `&& microdnf clean all` 
- without `&& microdnf clean all`: 334MB
- with `&& microdnf clean all`: 318MB
- without installing anything: 114MB

do you recommend alternative ways to install `curl` ?

Reply:
instead of a `kubectl exec curl` in the pod. you can do a [`client-go proxy service`](https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/service_expansion.go#L30)

Reply:
we can query the metrics server of the leader pod using the client-go directly

```
	result := o.clientset.CoreV1().RESTClient().Get().
		Namespace(pod.Namespace).
		Resource("pods").
		Name(fmt.Sprintf("%s:8383", pod.Name)).
		SubResource("proxy").
		Suffix("metrics").
		Do()
```

no need to install `curl` or exec in the pod.